cliquet/resource/__init__.py

```diff
@@ -1046,6 +1088,17 @@ def get_parent_id(self, request):
         """
         return ''
 
+    def _extract_fields(self):
+        """Override default fields extraction from QueryString to allow
+        the permission_field to exist.
+        """
+
+        fields = super(ShareableResource, self)._extract_fields()
+        if fields:
+            fields.append(self.model.permissions_field)
+
+        return fields
```

Comment:
This reminds me that @Natim suggested a while ago to hide the permissions when the user has not the permission to write the record.

I suggest that we handle this part in another pull-request if you don't mind :)